pyproject.toml

```diff
@@ -29,3 +29,11 @@ select = ["E", "F", "B", "SIM", "C4", "I"]
 ignore = ["E501", "E402", "C408", "SIM112"]
 line-length = 120
 target-version = "py38"
+
+[tool.mypy]
+packages = ["scripts"]
```

Comment:
here is where we tell mypy to only check the `scripts` package. note that we're using `packages = ["scripts"]` instead of `modules = ["scripts"]` because `packages` will recursive check subdirectories, while `modules` does not.

https://mypy.readthedocs.io/en/stable/config_file.html#confval-packages